### ISSUE TITLE
Difficulties installing & controlling devices using IrTrans

### ISSUE THREAD

**Issue by stefaniehoechtl** [asterics/AsTeRICS-Ergo#122]
AsTeRICS Ergo v4.0.0-alpha5
Microsoft Windows Vista Home Premium, VAIO Sony Corporation VGN-FW21M
I had difficulties with the installation of the IrTrans. When clicking on “Test”, the Test failed several times and I do not know why. Once I waited several minutes after opening “IRTransServer USB” and selected “Test” later – then the installation was successful. Afterwards I was able to set up the control of 1 device, but I was not able to control the device using IrTrans. I tried to add another device, but it was not recognised, that the IrTrans had already been installed. I closed AsTeRICS Ergo, disconnected and reconnected the IrTrans to the computer, closed and opened “IRTransServer USB” – then the control of the devices worked. Now everything works fine, it just wasn’t easy to get there.

**Comment by ChrisVeigl** [comment]
can you reproduce that problem (does it occur in the same way when do remove the IRTrans software) ?

**Comment by stefaniehoechtl** [comment]
Maybe I can get access to that laptop tomorrow and try again. I tested with 4 other laptops and never had that problem.

**Comment by stefaniehoechtl** [comment]
I tested again today. After opening "IRTransServer USB", I selected "Test", but the test failed. I selectet "Test" several times and after 1-2 min testing was successful. I guess we should just tell the users to be patient?
On some laptops, it also takes quite a long time after selecting the IrTrans setup file until a window pops up. Maybe we should tell them there as well that in some cases it just takes time?
